src/test/java/seedu/address/logic/commands/articlecommands/FilterArticleCommandTest.java

```diff
@@ -0,0 +1,67 @@
+package seedu.address.logic.commands.articlecommands;
+
+
+import static seedu.address.logic.commands.articlecommands.ArticleCommandTestUtil.assertCommandSuccess;
+import static seedu.address.testutil.TypicalArticles.getTypicalArticleBook;
+
+import java.time.LocalDateTime;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import seedu.address.logic.parser.ParserUtil;
+import seedu.address.logic.parser.exceptions.ParseException;
+import seedu.address.model.AddressBook;
+import seedu.address.model.Model;
+import seedu.address.model.ModelManager;
+import seedu.address.model.UserPrefs;
+import seedu.address.model.article.ArticleMatchesStatusPredicate;
+import seedu.address.model.article.ArticleMatchesTagPredicate;
+import seedu.address.model.article.ArticleMatchesTimePeriodPredicate;
+import seedu.address.model.article.PublicationDate;
+import seedu.address.model.tag.Tag;
+
+public class FilterArticleCommandTest {
+    private static final Tag EMPTY_TAG = null;
+    private static final PublicationDate EMPTY_START = new PublicationDate(LocalDateTime.MIN);
+    private static final PublicationDate EMPTY_END = new PublicationDate(LocalDateTime.MAX);
+    private Model model;
+    private Model expectedModel;
+    @BeforeEach
+    public void setUp() {
+        model = new ModelManager(new AddressBook(), getTypicalArticleBook(), new UserPrefs());
+        expectedModel = new ModelManager(new AddressBook(), model.getArticleBook(), new UserPrefs());
+    }
+    @Test
+    public void execute_zeroParameters_showUnfilteredList() throws ParseException {
+        assertCommandSuccess(new FilterArticleCommand("", EMPTY_TAG, EMPTY_START, EMPTY_END),
+                model, FilterArticleCommand.MESSAGE_SUCCESS, expectedModel);
+    }
+    //EP Only Status
+    @Test
+    public void execute_onlyStatus_showFilteredList() throws ParseException {
+        ArticleMatchesStatusPredicate predicate = new ArticleMatchesStatusPredicate("DRAFT");
+        expectedModel.updateFilteredArticleList(predicate);
+        assertCommandSuccess(new FilterArticleCommand("DRAFT", EMPTY_TAG, EMPTY_START, EMPTY_END),
+                model, FilterArticleCommand.MESSAGE_SUCCESS, expectedModel);
+    }
+    //ep Only tag
```

Comment:
just minor error, but ep should be EP..?

Reply:
Thanks for spotting, but i accidentally merged into this branch, so I can't really change now